### PR TITLE
[Symfony 5] Define APP_ENV during the build

### LINKED ISSUE #104: [Symfony] Make sure we are compatible with the 5.0 release
https://symfony.com/blog/symfony-5-0-0-released

### PR DESCRIPTION
**Pull request by EtienneM**
- `APP_ENV` is needed during the build otherwise the deployment fails with:

```
[...]
  - Installing symfony/yaml (v5.0.2): Loading from cache                                                                                                                                      
Generating optimized autoload files                                                                                                                                                           
       ocramius/package-versions: Generating version class...                                                                                                                                 
       ocramius/package-versions: ...done generating version class                                                                                                                            
Executing script cache:clear [KO]                                                                                                                                                             
 [KO]                                                                                                                                                                                         
Script cache:clear returned with error code 255                                                
!!  Symfony\Component\ErrorHandler\Error\ClassNotFoundError {#35                               
!!    #message: """                                                                            
!!      Attempted to load class "WebProfilerBundle" from namespace "Symfony\Bundle\WebProfilerBundle".\n                                                                                      
!!      Did you forget a "use" statement for another namespace?
!!      """                                                                                    
!!    #code: 0                                                                                 
!!    #file: "./src/Kernel.php"                                                                
!!    #line: 23                                                                                
!!    trace: {                                                                                 
[...]
```

Fix #104

### CODE REVIEW

File: conf/nginx/symfony4.conf.erb [hunk 1]
```diff
@@ -16,6 +16,7 @@ location ~ ^/index\.php(/|$) {
     fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
     fastcgi_param DOCUMENT_ROOT $realpath_root;
     fastcgi_param HTTPS off;
+    fastcgi_param APP_ENV <%= ENV['APP_ENV'] || 'prod' %>;
```
Comment (Soulou) [inline conf/nginx/symfony4.conf.erb]: I think this goes double with the environment variables:

https://symfony.com/doc/current/setup/web_server_configuration.html#apache-with-php-fpm

Comment (EtienneM) [reply conf/nginx/symfony4.conf.erb]: I don't understand the link you sent me.

But without it, the application does not work without specifying `APP_ENV=prod` here https://my.osc-fr1.scalingo.com/apps/sample-php-symfony5/variables

Comment (Soulou) [reply conf/nginx/symfony4.conf.erb]: But this is already injected here https://github.com/Scalingo/php-buildpack/pull/122/files#diff-c7b43338c09c3c9afd420fd8b6bb5198R410 (no need of extra var no?)

Comment (EtienneM) [reply conf/nginx/symfony4.conf.erb]: I thought so but apparently not as without this line the application does not work.

Comment (Soulou) [reply conf/nginx/symfony4.conf.erb]: Quite weird..

Comment (Soulou) [reply conf/nginx/symfony4.conf.erb]: But ok, it's not harmful anyway

**Pull request merged**